docs/DISCORD_FAQ.md

```diff
@@ -24,8 +24,8 @@ _Frequently Asked Questions_
 > **7: My Jacob Contest Display crops are wrong, how do I fix this?**
 > Run the command `/shclearcontestdata` to clear the Jacob contest data.
 
-> **8: How can I get bigger crop hit boxes?**
-> Use [Patcher](<https://sk1er.club/mods/patcher>) to have 1.12 hitboxes in 1.8.9.
+> **8: How can I get bigger crop hitboxes?**
+> Use [Polyfrost's PolyPatcher](<https://modrinth.com/mod/patcher>) (a fork of Patcher with more features and performance improvements) or [Sk1erLLC's Patcher](<https://sk1er.club/mods/patcher>) to have 1.12 hitboxes in 1.8.9.
```

Comment:
This feels a bit weirdly formatted. How about we leave the original sentence unchanged (use patcher to have ...) and then in two new lines link to patcher and poly patcher?